File: test/1/C/operand.c
#include <stdio.h>
#define bool int
#define true 1
#define false 0
bool a;
bool b;
int c;
int d;
float e;
float f;
char g;
char h;
int main()
{
a = true;
b = false;
c = 10;
d = 3;
e = 10.5;
f = 2.5;
g = 'a';
h = 'b';
printf("%s", "bool operand");
printf("%d", a && b);
printf("%d", a || b);
printf("%d",  !a);
printf("%d", a == b);
printf("%d", a != b);
printf("%s", "int operand");
printf("%d",  +c);
printf("%d",  -c);
printf("%d", c + d);
printf("%d", c - d);
printf("%d", c * d);
printf("%d", c * 1.0 / d);
printf("%d", c % d);
printf("%d", c / d);
printf("%d", c == d);
printf("%d", c != d);
printf("%d", c < d);
printf("%d", c > d);
printf("%d", c <= d);
printf("%d", c >= d);
printf("%s", "real operand");
printf("%f", e + f);
printf("%f", e - f);
printf("%f", e * f);
printf("%f", e * 1.0 / f);
printf("%d", e == f);
printf("%d", e != f);
printf("%d", e < f);
printf("%d", e > f);
printf("%d", e <= f);
printf("%d", e >= f);
printf("%s", "char operand");
printf("%d", g == h);
printf("%d", g != h);
printf("%s", "real int operand");
printf("%f", e + c);
printf("%f", e - c);
printf("%f", e * c);
printf("%f", e * 1.0 / c);
printf("%d", e == c);
printf("%d", e != c);
printf("%d", e < c);
printf("%d", e > c);
printf("%d", e <= c);
printf("%d", e >= c);
printf("%s", "int real operand");
printf("%f", c + e);
printf("%f", c - e);
printf("%f", c * e);
printf("%f", c * 1.0 / e);
printf("%d", c == e);
printf("%d", c != e);
printf("%d", c < e);
printf("%d", c > e);
}

File: test/1/Pascal/operand.pas
program TestOperator;
var
  a, b: boolean;
  c, d: integer;
  e, f: real;
  g, h: char;
begin
  a := true;
  b := false;
  c := 10;
  d := 3;
  e := 10.5;
  f := 2.5;
  g := 'a';
  h := 'b';

  write('bool operand');
  write(a and b);
  write(a or b);
  write(not a);
  write(a = b);
  write(a <> b);

  write('int operand');
  write(+c);
  write(-c);
  write(c + d);
  write(c - d);
  write(c * d);
  write(c / d);
  write(c mod d);
  write(c div d);
  write(c = d);
  write(c <> d);
  write(c < d);
  write(c > d);
  write(c <= d);
  write(c >= d);

  write('real operand');
  write(e + f);
  write(e - f);
  write(e * f);
  write(e / f);
  write(e = f);
  write(e <> f);
  write(e < f);
  write(e > f);
  write(e <= f);
  write(e >= f);

  write('char operand');
  write(g = h);
  write(g <> h);

  write('real int operand');
  write(e + c);
  write(e - c);
  write(e * c);
  write(e / c);
  write(e = c);
  write(e <> c);
  write(e < c);
  write(e > c);
  write(e <= c);
  write(e >= c);

  write('int real operand');
  write(c + e);
  write(c - e);
  write(c * e);
  write(c / e);
  write(c = e);
  write(c <> e);
  write(c < e);
  write(c > e);
end.